libbeat/common/config.go

```diff
@@ -327,33 +328,59 @@ func (f *flagOverwrite) Get() interface{} {
 	return f.value
 }
 
+func (ns *ConfigNamespace) Unpack(cfg *Config) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method ConfigNamespace.Unpack should have comment or be unexported